sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -446,6 +515,28 @@ internal final class URLSessionTransferManager: NSObject, TransferManager, URLSe
         self.transfer(transfer, didUpdateWithState: transfer.state, andProgress: nil)
     }
 
+    // MARK: Misc Methods
+
+    func handleNetworkTransition() {
+        let status = networkStatus
+        if let downloadPolicy = downloadConnectivityPolicy {
+            if downloadPolicy.transferOver.contains(status) {
```

Comment:
We should call `shouldTransfer` instead (here and below) for consistency.

Reply:
Done.